R/rd-data.R

```diff
@@ -160,6 +160,33 @@ as_data.tag_examples <- function(x, ...,
   paste(html, collapse = "")
 }
 
+example_fig_chunk_opts <- function(topic, obj_id) {
```

Comment:
There's no significant value to fetching from `fig_save`, but it doesn't prevent having two copies of the defaults. Specifying the values here might be cleaner.

Reply:
I.e., this could also just be:

``` r
example_fig_chunk_opts <- function(topic, obj_id) {
  list(
    dev = "png",
    dpi = 96,
    fig.retina = 2,
    dev.args = list(),
    fig.width = 7.291667,
    fig.height = 4.506593,
    fig.asp = 0.618047,
    fig.path = paste0("figures/", topic),
    # fig.cur is an internal chunk option that specifies
    # an index when there are multiple figures (i.e., per-topic).
    fig.cur = obj_id(topic)
  )
}